packages/assets/src/types.ts

```diff
@@ -26,14 +26,12 @@ export type Asset = {
   /** name of the asset */
   name: string;
   /** description of the asset */
-  description: string;
+  description?: string;
   /** symbol of the asset */
   symbol: string;
   /** icon of the asset */
-  icon: string;
+  icon: string | null;
   /** asset id on Fuel Network */
-  assetId: string;
```

Comment:
This assetId is the main id of the fuel network, this is why is outside of the folder

Reply:
the id can be different on different networks